crates/iota/src/name_commands.rs

```diff
@@ -527,7 +541,9 @@ impl NameCommand {
                 opts,
             } => {
                 // Check ownership of the name off-chain to avoid potentially wasting gas
-                get_proxy_nft_by_name(&name, context).await?;
+                let sender =
+                    get_identity_address(opts.rest.sender.map(Into::into), context).await?;
+                get_proxy_nft_by_name(&name, Some(sender), context).await?;
```

Comment:
Why do you call `get_identity_address()` before and provide `Some(sender)` instead of just `sender` as in the other places?

Reply:
because it's used later